tests/integration/client/feedback/training/test_trl.py

```diff
@@ -0,0 +1,256 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from collections import Counter
+from typing import TYPE_CHECKING, Any, Dict, Iterator, List
+
+import pytest
+from argilla.client.feedback.dataset import FeedbackDataset
+from argilla.client.feedback.schemas.records import FeedbackRecord
+from argilla.client.feedback.training.base import ArgillaTrainer
+from argilla.client.feedback.training.schemas import TrainingTask
+from datasets import Dataset, DatasetDict
+
+from tests.integration.training.helpers import train_with_cleanup
+
+if TYPE_CHECKING:
+    from argilla.client.feedback.types import AllowedFieldTypes, AllowedQuestionTypes
+
+
+@pytest.mark.usefixtures(
+    "feedback_dataset_guidelines",
+    "feedback_dataset_fields",
+    "feedback_dataset_questions",
+    "feedback_dataset_records",
+)
```

Comment:
We should refactor the tests to use the pre-defined factories rather than those fixtures, but that can be tackled in a separate PR 👍🏻

Reply:
These are normal fixtures, no idea why the weird `usefixtures` syntax is used, it's not needed at all.

Reply:
7e2c28eb14688ab8898c370462a76e5783d826c7